IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcGroundReinforcementElement/DocEntity.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcGroundReinforcementElement" UniqueId="eff9d345-72ca-4b29-8857-cc24f5c55fb0" BaseDefinition="IfcBuiltElement" EntityFlags="32">
+	<Attributes>
+		<DocAttribute Name="PredefinedType" UniqueId="5bf261d3-adfb-4e71-89d3-59707720d1ca" DefinedType="IfcGroundReinforcementElementTypeEnum">
+			<Documentation>Identifies the predefined type of ground reinforcementl elements. This type may associate additional specific property sets.</Documentation>
+		</DocAttribute>
+	</Attributes>
+	<WhereRules>
+		<DocWhereRule Name="CorrectPredefinedType" UniqueId="566bf0f8-aaeb-41ca-8145-127664b1dbee">
+			<Documentation>Either the _PredefinedType_ attribute is unset, or the inherited attribute _ObjectType_ shall be provided, if the _PredefinedType_ is set to USERDEFINED.</Documentation>
+			<Expression>NOT(EXISTS(PredefinedType)) OR
```

Comment:
Probably should use HasObjectType rule. PredefinedType looks mandatory already.

Reply:
Should be fixed now.